backend/qemu.pm

```diff
@@ -244,13 +244,14 @@ sub start_qemu {
         $vars->{BIOS} //= $vars->{UEFI_BIOS};
     }
 
-    if ($vars->{BIOS} && $vars->{BIOS} !~ /^\//) {
-        # Non-absolute paths are assumed relative to /usr/share/qemu
-        $vars->{BIOS} = '/usr/share/qemu/' . $vars->{BIOS};
-    }
-
-    if ($vars->{BIOS} && !-e $vars->{BIOS}) {
-        die "'$vars->{BIOS}' missing, check BIOS\n";
+    foreach my $attribute (qw/BIOS KERNEL INITRD/) {
+        if ($vars->{$attribute} && $vars->{$attribute} !~ /^\//) {
```

Comment:
I don't understand what do you mean by that. Do you mean normalizing path like Python's [normpath](https://docs.python.org/2/library/os.path.html#os.path.normpath)?